### DISCUSSION TITLE
Next Steps for vNext

### DISCUSSION THREAD

**Discussion by jellypuno** [zowe/zowe-explorer-vscode#1412] [Squad Discussion]
For Zowe Explorer V2, we are targeting for the following:
1. Approved v1 Conformance Criteria
2. Support for team configuration where users can create their own initial team-config file via ZE and load existing team-config file created via CLI

Once this MVP is released and available, we want to implement the following
1. v2 Conformance Criteria
2. Management of team config (Update and Delete)

I am opening this discussion so we could have a channel to talk about our next steps regarding team-config. We have lots of open questions and it would be great if we could start discussing them now.

Open Questions:
1. Are we going to allow updating of the team-config via VSCode dialog boxes or Web Views? Right now, we are opening the team-config file and we are asking users to refresh VSCode so the changes will take effect. This is a workaround and we would need a better solution for this
2. Are we going to handle multi-root workspace of team-config?
3. How about tokens?
4. Are we deprecating our old profile management? (creating of yaml files)

**Comment by zFernand0** [comment]
V2 has been released and the Conformance Criteria is solidified.
As for the Open Questions:
1. Delete and Update will just open the Team Profile. We can revisit if a user complains about it in the future 😋 
2. Multiple issues have been opened about this, #1226, #1741 
3. Tokens should still be supported with Team Profiles 😋 
4. We don't have to deprecate v1 profiles just yet. However we do recommend people to migrate to Team Profiles. We can consider deprecation/removal for Zowe V3
